t/apicast-policy-rate-limit.t

```diff
@@ -1169,3 +1169,64 @@ so only the third call returns 429.
 [200, 200, 200, 429]
 --- no_error_log
 [error]
+
+=== TEST 19: Rejected (count).
```

Comment:
Can you mention in the title or the description that this is testing a config with multiple limiters of the same type?
Without that, it's difficult to know what this test is testing that all the rest are not.

Reply:
Fixed.